app/components/Quest/LoadingScreen.tsx

```diff
@@ -0,0 +1,9 @@
+import Loading from "./Loading"
+
+export default function LoadingScreen({ color="sky" }) {
```

Comment:
We should check with Allan to use a color from figma.

Reply:
same applied the classic bg-green for now.